client/src/views/SpecificJobPage/SpecificJobPage.tsx

```diff
@@ -0,0 +1,314 @@
+      },
+    ])
+
+    setCompanyInfo({
+      companyName: job?.companyName ?? '',
+      city: job?.city ?? '',
+      country: job?.country ?? '',
+      positionTitle: job?.jobName ?? '',
+    })
+
+    setExtraInfo([
+      {
+        title: 'Deadline',
+        text: job?.appDeadline ?? '',
+      },
+      {
+        title: 'Compensation/Benefits',
+        text: job?.compensationAndBenefitsInformation ?? '',
+      },
+    ])
+  }, [job])
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://842gb0w279.execute-api.ca-central-1.amazonaws.com/items/${companyInfo.companyName}`,
+      )
+      .then((res: any) => {
+        if (res.data.Item) {
+          setImageURL(res.data.Item.logo)
+
+          const dashIndex = res.data.Item.salary.indexOf('-')
+          let salary = `$${res.data.Item.salary}`
+          if (dashIndex >= 0)
+            salary = `${salary.slice(0, dashIndex + 2)}$${salary.slice(
+              dashIndex + 2,
+            )}`
+          if (res.data.Item.Currency) salary += ` ${res.data.Item.Currency}`
+
+          setJobInfo([
+            ...jobInfo,
+            {
+              title: 'Salary',
+              text: salary,
+            },
+            {
+              title: 'Company Website',
+              text: `<a href=//${res.data.Item.domain} target='_blank'>${res.data.Item.domain}</a>`,
+            },
+          ])
+        } else {
+          setImageURL(defaultIcon)
+        }
+      })
+      .catch((err: any) => err)
+  }, [companyInfo, jobInfo])
+
+  useEffect(() => {
+    document.title = `Wave - ${job?.jobName}`
+  }, [job])
+
+  // const Search = () => (
+  //   <>
+  //     <Spacer />
+  //     <Typography color={primary} variant="h5" fontWeight="bold">
+  //       Explore Jobs
+  //     </Typography>
+  //     <Spacer />
+  //     {/* <SearchBar width="50%" minWidth="300px" /> */}
+  //     <Spacer />
+  //   </>
+  // )
+
+  const CompanyHeader = () => (
+    <>
+      {imageURL !== '' && (
+        <>
+          <Grid item xs={12} md={8}>
+            <CompanyCard
+              imageURL={imageURL}
+              companyName={companyInfo.companyName}
+              city={companyInfo.city}
+              country={companyInfo.country}
+              positionTitle={companyInfo.positionTitle}
+            />
+          </Grid>
+        </>
+      )}
+    </>
+  )
+
+  const Description = () => (
+    <Box style={{ marginLeft: '110px', maxWidth: '60%', padding: '16px' }}>
+      {jobInfo &&
+        jobInfo.map((item: { title: string; text: string }) => {
+          return (
+            item.text && (
+              <SpecificJobPageSection
+                jobSectionTitle={item.title}
+                jobSectionDescription={ReactHtmlParser(item.text)}
+              />
+            )
+          )
+        })}
+    </Box>
+  )
+  const StyledTab = styled(Tab)({
+    '&.MuiTab-textColorPrimary': {
+      color: 'white',
+    },
+  })
+  const StyledShortlistButton = styled(MUIButton)({
```

Comment:
are other buttons in our design looking the same?
In general, I think we should probably start creating reuseable components when applicable